libmpv/cocoa-openglcb/cocoa-openglcb.m

```diff
@@ -179,19 +195,23 @@ - (void) applicationDidFinishLaunching:(NSNotification *)notification {
 
     check_error(mpv_initialize(mpv));
     check_error(mpv_set_option_string(mpv, "vo", "opengl-cb"));
-    mpv_opengl_cb_context *mpvGL = mpv_get_sub_api(mpv, MPV_SUB_API_OPENGL_CB);
-    if (!mpvGL) {
-        puts("libmpv does not have the opengl-cb sub-API.");
+    
+    mpv_render_param params[] = {
+        {MPV_RENDER_PARAM_API_TYPE, MPV_RENDER_API_TYPE_OPENGL},
+        {MPV_RENDER_PARAM_OPENGL_INIT_PARAMS, &(mpv_opengl_init_params){
+            .get_proc_address = get_proc_address,
+        }},
+        {MPV_RENDER_PARAM_INVALID, 0}
+    };
+    mpv_render_context *mpvGL;
+    if (mpv_render_context_create(&mpvGL, mpv, params) < 0) {
+        puts("libmpv does not obtain the render context.");
```

Comment:
this sounds a bit weird. `libmpv couldn't create a render context` or along the lines is probably more appropriate.

Reply:
Okay, how about changing to `failed to initialize mpv GL context`? It also described in [qml example snippets](/mpv-player/mpv-examples/blob/e72572b7b2bae5f3512ff7f188ce0e99a4383c42/libmpv/qml/main.cpp)